spiceaidocs/docs/data-accelerators/index.md

```diff
@@ -63,6 +63,17 @@ datasets:
         SELECT * FROM accelerated_dataset WHERE city = 'Seattle'
 ```
 
+`refresh_sql` can be updated on-demand via `PATCH /v1/datasets/:name/acceleration` API endpoint. This modification is non-persistent and will revert to the original value at the next runtime restart. An example query using cURL:
```

Comment:
Let's add a warning / limitations section here so its very apparent that its not persistent.

Reply:
Thank you - added to section limitations (last point)
<img width="678" alt="image" src="https://github.com/spiceai/docs/assets/981580/88b3c731-73f2-4dd8-a6d1-cf5ceb1e9d81">